src/config/network/index.ts

```diff
@@ -35,6 +51,48 @@ const networkMap: INetworkMap = {
     },
     rpcUrls: ['https://matic-mumbai.chainstacklabs.com']
   },
+  [ChainIDEnum.HECO]: {
+    chainId: '0x80',
+    blockExplorerUrls: ['https://hecoinfo.com/'],
+    chainName: 'Heco Chain',
+    iconUrls: [
+      'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6f259637dcD74C767781E37Bc6133cd6A68aa161/logo.png'
+    ],
+    nativeCurrency: {
+      name: 'Huobi ECO Chain Token',
+      symbol: 'HT',
+      decimals: 18
+    },
+    rpcUrls: ['https://http-mainnet.hecochain.com']
+  },
+  [ChainIDEnum.HECO_TEST]: {
+    chainId: '0x100',
+    blockExplorerUrls: ['https://testnet.hecoinfo.com/'],
+    chainName: 'Heco Testnet',
+    iconUrls: [
+      'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6f259637dcD74C767781E37Bc6133cd6A68aa161/logo.png'
+    ],
+    nativeCurrency: {
+      name: 'Huobi ECO Chain Token',
+      symbol: 'HT',
+      decimals: 18
+    },
+    rpcUrls: ['https://http-testnet.hecochain.com']
+  },
+  [ChainIDEnum.POLYGON]: {
+    chainId: '0x89',
+    blockExplorerUrls: ['https://polygonscan.com/'],
+    chainName: 'Polygon Chain',
+    iconUrls: [
+      'https://alphaevolution.l2me.com/assets/chain/polygon/matic.png'
```

Comment:
https://github.com/evolutionlandorg/evolution-interface/commit/6e7b1b240315a931c88826942a1e89d36cbb6694

The l2me domain name has expired. Modify iconUrls according to this commit